### DISCUSSION TITLE
Usage and applications

### DISCUSSION THREAD

**Discussion by great9** [cariboulabs/cariboulite#140] [General]
I'm curious to know what are you guys using your cariboulite for?

**Comment by K7MDL2** [comment]
Been using it mostly with OpenWebRx+ while traveling and aboard ship where bandwidth is limited.  Listen to regular ham stuff (VHF SSB weak signal weekly nets and HF FT8 monitoring) and marine weather and AIS messages.  Have 2x HL2, the CaribouLite and RTL-SDR dongle served by the RPi4B for plenty of band coverage.

**Reply by great9** [reply]
How did you set it up?
Added it as a new device "RTL-SDR through SoapySDR"?
Any other config options?

**Comment by K7MDL2** [comment]
Was last spring, I forget the low level details.  It was probably Soapy though.   It is not plugged in at the moment to do a SoapySDRUtil -probe check.
Check out openwebrx (rtl-sdr.com)<https://www.rtl-sdr.com/tag/openwebrx/> and Mare & Gal Electronics > Blog Archive > Multiband SDR with remote receivers (pavlin.si)<https://e.pavlin.si/2021/12/11/multiband-sdr-with-remote-receivers/>



***@***.***

For the HL2, you need to create a new device with 1 profile for each band to share its receivers.  Otherwise the last person to choose a profile changes the frequency for all users on that device.  That is why there is a HF HL2 and HF HL2 80M device below.

***@***.***
